core/templates/dev/head/exploration_editor/EditorServices.js

```diff
@@ -673,6 +673,65 @@ oppia.factory('explorationStatesService', [
       editorContextService, validatorsService, newStateTemplateService,
       explorationGadgetsService) {
     var _states = null;
+
+    // Maps backend names to the corresponding frontend dict accessor lists.
+    var PROPERTY_REF_DATA = {
```

Comment:
Not sure. It only gets used here, so it seems to me like it's better for it to be a local thing...